src/metax_api/services/catalog_record_service.py

```diff
@@ -760,3 +762,22 @@ def get_research_dataset_license_url(rd):
         license = rd['access_rights']['license'][0]
 
         return license.get('identifier') or license.get('license')
+
+    @classmethod
+    def destroy_bulk(cls, cr_identifiers):
+        """
```

Comment:
This function signature could be written with type hints:
```python
from typing import List
...
def destroy_bulk(cls, cr_identifiers:List[int])
```

Reply:
never mind, I don't know if those values are ints.

Reply:
yeah, these can be ints and strings, but further in code is similar check